plot-api/src/main/kotlin/jetbrains/letsPlot/scale/DateTime.kt

```diff
@@ -12,7 +12,23 @@ import jetbrains.datalore.plot.config.Option.Scale.DATE_TIME
 import jetbrains.letsPlot.intern.Options
 import jetbrains.letsPlot.intern.Scale
 
-
+/**
+ * Continuous position scale (x).
+ *
+ * @param name string
+ *      The name of the scale - used as the axis label or the legend title. If None, the default, the name of the scale
+ *      is taken from the first mapping used for that aesthetic.
+ * @param breaks list of numeric
+ *     A numeric vector of positions (of ticks).
+ * @param labels list of strings
+ *      A vector of labels (on ticks).
+ * @param limits list of numerics
+ *      A numeric vector of length two providing limits of the scale.
+ * @param expand
+ *      A numeric vector of length two giving multiplicative and additive expansion constants.
```

Comment:
Document the defaults:
- continuous  scale: multiplicative = 0.05, additive = 0
- discrete scale: multiplicative= 0, additive = 0.6 
see: https://ggplot2.tidyverse.org/reference/scale_continuous.html

Reply:
Info about default values was added. Also need to add this link to the description?

Reply:
No the link is mot needed.